src/app/features/event/single-event.component.ts

```diff
@@ -0,0 +1,24 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { SingleEventService } from './single-event.service';
+import { Store } from '@ngrx/store';
+import { AppState } from 'src/app/app.module';
+
+@Component({
+  selector: 'app-single-event',
+  templateUrl: './singleEvent.component.html',
+  styleUrls: ['./singleEvent.component.scss'],
+  providers: [SingleEventService],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SingleEventComponent {
+  private singleEventService = inject(SingleEventService);
+  private store = inject<Store<AppState>>(Store);
+  private eventId = '1';
```

Comment:
dlaczego mamy wyjściowo takie id ustawione?

Reply:
bo będzie ono przekazywane z innego komponentu, ale jeszcze tego nie zrobiliśmy w kodzie więc na ten moment jest ustawione na sztywno.

Reply:
prośba w takim razie o dodawnie koemntarza do tego wraz z taskiem który ma to poprawić :) najlepiej jako link do danego taska